src/status_im/chat/handlers/input.cljs

```diff
@@ -332,3 +337,47 @@
   (fn [{:keys [current-chat-id] :as db} [_ text chat-id]]
     (let [chat-id (or chat-id current-chat-id)]
       (assoc-in db [:chats chat-id :seq-argument-input-text] text))))
+
+(handlers/register-handler
+  :update-text-selection
+  (handlers/side-effect!
+    (fn [{:keys [current-chat-id] :as db} [_ selection]]
+      (let [input-text (get-in db [:chats current-chat-id :input-text])
+            command    (input-model/selected-chat-command db current-chat-id input-text)]
+        (when (and (= selection (+ 2 (count (get-in command [:command :name]))))
```

Comment:
@alwx it's the code is not super clear here, could you add some comment? I mean this `+ 2` needs some explanation, `(when what-exactly? ...)`